Makefile

```diff
@@ -1,89 +1,47 @@
 export GO15VENDOREXPERIMENT := 1
 
-UNAME := $(shell uname)
-ifeq ($(UNAME), Darwin)
-    HASH := md5
-else
-    HASH := md5sum
-endif
-
 all: build manifests
 
-generate: sync
-	find pkg/ -name "*generated*.go" -exec rm {} -f \;
-	./hack/build-go.sh generate ${WHAT}
-	goimports -w -local kubevirt.io cmd/ pkg/ tests/
-	./hack/bootstrap-ginkgo.sh
-	(cd tools/openapispec/ && go build)
-	tools/openapispec/openapispec --dump-api-spec-path api/openapi-spec/swagger.json
+generate:
+	hack/dockerized "hack/glide-checksync.sh && ./hack/generate.sh"
 
-apidocs: generate
-	./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html
+apidocs:
+	hack/dockerized "hack/glide-checksync.sh && ./hack/generate.sh && ./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html"
 
-build: checksync fmt vet compile
+build:
+	hack/dockerized "hack/glide-checksync.sh && ./hack/check.sh && ./hack/build-go.sh install ${WHAT}"
 
 goveralls:
-	./hack/goveralls.sh
-
-compile:
-	./hack/build-go.sh install ${WHAT}
-
-vet:
-	./hack/build-go.sh vet ${WHAT}
-
-fmt:
-	goimports -w -local kubevirt.io cmd/ pkg/ tests/
+	hack/dockerized ./hack/goveralls.sh
 
-fmt-bash:
-	shfmt -i 4 -w cluster/ hack/ images/
-
-test: build
-	./hack/build-go.sh test ${WHAT}
+test:
```

Comment:
Just tried running `make test` and had a failure due to missing vendor. Looks like this target needs to depend on sync.

Reply:
And they definitely depend on build too (pkg/virt-handler/monitor_test.go at least).

Reply:
Done.